apps/web/modules/settings/platform/oauth-clients/[clientId]/edit/edit-webhooks-view.tsx

```diff
@@ -77,6 +77,14 @@ export default function EditOAuthClientWebhooks() {
                     value: WebhookTriggerEvents.AFTER_GUESTS_CAL_VIDEO_NO_SHOW,
                     label: "after_guests_cal_video_no_show",
                   },
+                  {
```

Comment:
Should these be separate per video type? Seems like it should be `AFTER_HOSTS_NO_SHOW` and `AFTER_GUESTS_NO_SHOW` no matter which video type they used.